data/test/scenarios/lua_tests/wesnoth/test_wfl_rotate_loc_around.cfg

```diff
@@ -0,0 +1,12 @@
+{GENERIC_UNIT_TEST "test_wfl_rotate_loc_around" (
```

Comment:
tests are expected to have description comments

Reply:
I wasn't sure what to write there